show/main.py

```diff
@@ -2434,6 +2434,211 @@ def tunnel():
 
     click.echo(tabulate(table, header))
 
+#
+# 'system-health' command ("show system-health")
+#
+@cli.group(name='system-health', cls=clicommon.AliasedGroup)
```

Comment:
We are currently in the process of making the show/config code more manageable by moving groups into separate files, thus keeping the main.py file smaller. Can you please move this new `system-health` group into a separate file, `system_health.py`? Please use `vlan.py` as an example.

Reply:
Fixed, please check.